arbitrum-docs/node-running/how-tos/local-dev-node.mdx

```diff
@@ -37,18 +33,51 @@ To relaunch the node after the first installation, run the following command.
 
 ⚠️ Note that running with the --init flag will clear all chain data and redeploy!
 
+### Additional arguments
+
+You can find a list of additional arguments to use with `test-node.bash` by using `--help`.
+
+```bash
+./test-node.bash --help
+```
+
+### Helper scripts
+
+The repository includes a set of helper scripts for basic actions like funding accounts or bridging funds. You can see a list of the available scripts by running:
```

Comment:
I think the one script most users would use is funding an address. I suggest adding a direct example for that here that will require minimal reading/expirementation to fit to their needs.

should be something like

```
./test-node.bash script send-l2 --to address_0x11223344556677889900 --ethamount 5
```

Reply:
Great idea. Added.
Thanks!